packages/evolution-legacy/src/components/survey/Survey.js

```diff
@@ -204,6 +204,7 @@ export class Survey extends React.Component {
                   submitted                   = {this.props.submitted}
                 />
               </form>
+              <div dangerouslySetInnerHTML={{__html: this.props.t(`survey:footer`)}} />
```

Comment:
To avoid surveys who do not have this string defined to have the text `survey:footer` at the bottom, you can do `props.t(['survey:footer', 'main:footer'])`, define the `footer` string in evolution's `main.json` file in the locale (it can be empty).

For the example in the demo_survey, you could use more generic links than the artm's own (this project is open source and could be used by anyone in the world). For example, a link to evolution's github page!

Reply:
Good points! Fixed :)